content/en/synthetics/guide/synthetic-test-monitors.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using Synthetic Test Monitors
+kind: guide
+description: Learn about Synthetic monitors created with your Synthetic tests. 
+further_reading:
+- link: "/monitors/manage/"
+  tag: "Documentation"
+  text: "Manage Monitors"
+- link: "/synthetics/metrics/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Metrics"
+---
+
+## Overview
+
+When you create a Synthetic test, Datadog automatically creates an associated monitor. You are notified when the Synthetic test monitor alerts.
+
+{{< img src="synthetics/guide/synthetics_test_monitors/synthetic_test_monitor.png" alt="Synthetic test monitor" style="width:100%;">}}
+
+## Create a Synthetic test monitor
+
+<div class="alert alert-info">You cannot create or import a Synthetic test monitor in <a href="/monitors/create/">Monitors</a>.</div>
+
+Create a monitor in the **Configure the monitor for this test** section to send notifications when a Synthetic test is failing. Monitors are associated with the Synthetic test you create and set alerting conditions for. 
+
+{{< img src="synthetics/guide/synthetics_test_monitors/configure_the_monitor_for_this_test.png" alt="Creating a monitor in your Synthetic test" style="width:90%;">}}
+
+Customize the monitor name to search for it on the [**Manage Monitors**][1] page. You can use [conditional variables][2] to characterize the notification message based on test state. To find a Synthetic test monitor, filter on `type:synthetics` in the search bar.
+
+The Synthetic test monitor integrates with notification channels such as email, Slack, Pagerduty, and Microsoft Teams. For more information, see [Notifications][3].
+
+Synthetic test monitors do not support tag and attribute variables or the `{{#is_warning}}`, `{{^is_warning}}`, `{{#is_warning_recovery}}`, and `{{^is_warning_recovery}}` conditional variables in the the monitor name. To use tag and attribute variables, create a [metric monitor][4].
```

Comment:
Did we want to remove the explicit variables we don't support? And just have something like:

```suggestion
To use Monitor attribute and tag variables, create a [metric monitor][4].
```

And do we want to link to the separate guide for using the `synthetics.test_runs` metric?

Reply:
Yes, we will do the latter in DOCS-3513.